.bcr/presubmit.yml

```diff
@@ -0,0 +1,12 @@
+bcr_test_module:
+  module_path: "examples"
+  matrix:
+    platform: ["debian10", "macos", "ubuntu2004", "windows"]
+    bazel: ["7.x", "6.x"]
```

Comment:
Does it make sense to add rolling? Bazel 8 should be coming soon?

Reply:
yes - but I'm guessing that will be red to start with, so maybe it should be a follow-up PR?
This BCR presubmit has a downside that it runs only when we push a release, so there's no signal on this PR to predict what will happen.

Reply:
Makes sense